google/ggrc-core
# GGRC-8256: User should not receive a message... Fix regression.

# Issue description

My implementation of GGRC-8256 affected imports of all objects linked with issue tracker. Fixed that.

# Steps to test the changes

1) Import assessment linked to issue tracker;
2) Notice that email which says that related tickets in issue tracker were updated is using bulk verify specific messages.

# Solution description

While processing tasks from task queue use custom messages for bulk verify tasks.

# Sanity checklist


# PR Review checklist
